content/shmem_barrier_all.tex

```diff
@@ -42,6 +44,12 @@
     The \FUNC{shmem\_barrier\_all} routine can be used to
     portably ensure that memory access operations observe remote updates in the order
     enforced by initiator PEs.
+
+    \newtext{The \FUNC{shmem\_barrier\_all} routine does not ensure completion
+    of operations performed on contexts with the \CONST{SHMEM\_CTX\_PRIVATE}
+    option enabled.  Calls to \FUNC{shmem\_ctx\_quiet} can be performed prior
+    to calling the barrier routine to ensure completion of operations issued on
+    private contexts.}
 }
```

Comment:
Given that the semantic has been changed wrt shared-contexts, I feel like this small paragraph needs to be adapted as well.